doc/ChangeLog

```diff
@@ -1,4 +1,98 @@
 ===============================================================
+Tag name:  ctsm1.0.dev027
+Originator(s):  negins (Negin Sobhani,UCAR/CSEG,303-497-1224)
+Date: Tue Feb 19 12:57:12 MST 2019
+One-line Summary: Non-constant time initialization for soil hydrology types are moved to SoilHydrologyType.F
+
+Purpose of changes
+------------------
+
+[Fill this in with details]
+Initalization of time-varying `zwt`, `zwt_perched`, and `frost_table` are moved to `InitCold` in `SoilHydrologyType`.
+
+Previously, `zwt`, `zwt_perched`, and `frost_table` were initialized in `SoilHydrologyInitTimeConstMod`.
+
+Bugs fixed or introduced
+------------------------
+Issues fixed (include CTSM Issue #575)
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+None
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_0
+
+[ ] clm4_5
+
+Notes of particular relevance for users
+---------------------------------------
+
+Caveats for users (e.g., need to interpolate initial conditions):
+
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+
+Changes made to namelist defaults (e.g., changed parameter values):
+
+Changes to the datasets (e.g., parameter, surface or initial files):
+
+Substantial timing or memory changes: [For timing changes, can check PFS test(s) in the test suite]
+
+Notes of particular relevance for developers: (including Code reviews and testing)
+---------------------------------------------
+NOTE: Be sure to review the steps in ../CTSMMasterChecklist as well as the coding style in the Developers Guide
+
+Caveats for developers (e.g., code that is duplicated that requires double maintenance):
```

Comment:
Again, please answer this and the next question ("none" is fine).

Reply:
Thanks. Fixed.